pkg/exchange/ftx/stream.go

```diff
@@ -102,15 +117,69 @@ func (s *Stream) SetPublicOnly() {
 	atomic.StoreInt32(&s.publicOnly, 1)
 }
 
-func (s *Stream) Subscribe(channel types.Channel, symbol string, _ types.SubscribeOptions) {
-	if channel != types.BookChannel {
-		panic("only support book channel now")
+func (s *Stream) Subscribe(channel types.Channel, symbol string, option types.SubscribeOptions) {
+	if channel == types.BookChannel {
+		s.addSubscription(websocketRequest{
+			Operation: subscribe,
+			Channel:   orderBookChannel,
+			Market:    toLocalSymbol(TrimUpperString(symbol)),
+		})
+
+	} else if channel == types.KLineChannel {
+		// FTX does not support kline channel, do polling
+		go s.pollKLines(symbol, option)
```

Comment:
we should still add the kline subscription to the subscription list.

and move `go s.pollKLines(symbol, option)` to the Connect() method.